docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -405,3 +405,5 @@ def my_asset(context):
     # fetches contents of an asset
     return context.resources.api.fetch_table("my_asset")
 ```
+## Further Reading
+Interested in learning more about software-defined assets and working through a more complex example? Check out our [guide on software-defined assets](/guides/dagster/software-defined-assets)
```

Comment:
I think it's worth including a link to the modern data stack asset example as well.

Also, nitpick: missing period at end of sentence.

Reply:
done!